src/context/ConstantContext.tsx

```diff
@@ -8,19 +8,21 @@ import {
 
 import { ConfigureSessionTypes } from '@uikit/lib/hooks/useConnect/types';
 import { LabelStringSet } from '@uikit/ui/Label';
+import { StringSet } from '@uikit/ui/Label/stringSet';
 
 import { type Constant, DEFAULT_CONSTANT, widgetStringSet } from '../const';
 import { getDefaultServiceName, isMobile } from '../utils';
 
 const initialState = DEFAULT_CONSTANT;
 
-interface ConstantContextProps extends Partial<Constant> {
-  applicationId: string | null;
-  botId: string | null;
+interface ConstantContextProps extends Omit<Partial<Constant>, 'stringSet'> {
```

Comment:
what is goal of omit stringSet?

Reply:
to avoid type error

Reply:
no i meant since stringSet is already in Constant why omit here and add to ConstantContextProps?

Reply:
Currently, the `ConstantContextProps` type takes the `Partial<StringSet>` type.
However, since `StringSet` is declared in the `Constant` type, it should be omitted, and the `Partial<StringSet>` type should be added to avoid type errors.

Reply:
ah you Partial => optional. got it, thanks!